app/src/ui/preferences/appearance.tsx

```diff
@@ -119,14 +133,25 @@ export class Appearance extends React.Component<
       <DialogContent>
         <h2 id="theme-heading">Theme</h2>
 
-        <RadioGroup<ApplicationTheme>
-          ariaLabelledBy="theme-heading"
-          className="theme-selector"
-          selectedKey={selectedTheme}
-          radioButtonKeys={themes}
-          onSelectionChanged={this.onSelectedThemeChanged}
-          renderRadioButtonLabelContents={this.renderThemeSwatch}
-        />
+        <Row>
+          <RadioGroup<ApplicationTheme>
+            ariaLabelledBy="theme-heading"
+            className="theme-selector"
+            selectedKey={selectedTheme}
+            radioButtonKeys={themes}
+            onSelectionChanged={this.onSelectedThemeChanged}
+            renderRadioButtonLabelContents={this.renderThemeSwatch}
+          />
+        </Row>
+
+        <h2>Title bar style</h2>
+        <Select
+          value={this.state.titleBarStyle}
+          onChange={this.onSelectChanged}
+        >
+          <option value="native">Native</option>
+          <option value="custom">Custom</option>
+        </Select>
```

Comment:
🤔 it'd be great if we could add some context below the input here based on the option you choose here, but I'm not sure what words we want to add here:

![Screenshot from 2023-07-29 11-01-05](https://github.com/shiftkey/desktop/assets/359239/6bd0a78f-c28e-43af-8674-d1a0406c3182)

The limited space here could be an issue, but visually I could see it looking like the grey text below this checkbox:

![Screenshot from 2023-07-29 11-00-36](https://github.com/shiftkey/desktop/assets/359239/a1551f88-1ea9-4d3b-bcee-25b8d5faac72)

Some suggested words to kick this idea along:

> **Native**
>
> Uses the menu system and chrome provided by your window manager

> **Custom**
>
> Uses the menu system provided by GitHub Desktop, hiding the default chrome provided by your window manager.